### ISSUE TITLE
Dont allow anon users to download or even see attachments

### ISSUE THREAD

**Issue by SofttouchCode** [mimecorg/webissues#78]
Currently, anon users can see issues, comments and attachments (and can download them). I believe, there should be a setting about what anon user can do, so we can prevent downloading attachments.

**Comment by mimecorg** [comment]
Anonymous users can only view issues, comments and attachments in public projects. Regular projects (which is the default setting for all projects) are not accessible to anonymous users.

**Comment by SofttouchCode** [comment]
Yes, but would it not make sense when anon user can only see things, but not download attachments? Sometimes, there is a project what should be visible to the public so they see the progress of something, but they still should not be allowed to download attachments, only after they have registered. Everything else it just great.
